content/sensu-go/5.20/dashboard/filtering.md

```diff
@@ -202,9 +202,10 @@ To save a filtered search:
 4. Type the name you want to use for the saved search.
 5. Press **Return/Enter**.
 
-Any Sensu user with access to the namespace where you save the search will be able to recall it at any time.
-
 Sensu saves your filtered searches to etcd in a resource named `searches`.
+The `searches` resource is a [namespaced resource type][11], so any Sensu user assigned to a [role][12] that includes permissions for the namespace where you save the search will be able to recall it at any time.
```

Comment:
This might be a bit misleading, or maybe I just misunderstood it, but you need explicit permissions on the `searches` resource, unless you have access to all resources on a given namespace.

Reply:
thanks Simon -- I had a feeling I was off but I wasn't sure how. I'll get it updated.